src/django/philly/templates/philly/map.html

```diff
@@ -0,0 +1,38 @@
+{% extends "philly/base.html" %}
+
+{% block content %}
+
+<h1>Philadelphia neighborhoods </h1>
+ <div id="mapid"></div>
+<style>
+    #mapid {
+        height: 600px;
+        margin: auto;
+        width: 800px;
+    }
+    h1 {
+
+    }
+</style>
+<script type="text/javascript">
+    var gj = '{{ data|escapejs }}';
```

Comment:
Sorry for not catching this on the previous review, but shouldn't this be a Javascript object? Is there a reason that this has to be inserted as a string and then parsed?

Reply:
I seem to be getting an error when I run without the escapejs filter:

`Unexpected token & in JSON at position 1`

Reply:
Sorry, the `escapejs` wasn't the thing that I was confused about, but rather why this `var gj = '{{ data|escapejs }}'` rather than `gj = {{ data|escapejs }};`. Then you could eliminate the `parseJSON()` (or `JSON.parse()`) call below entirely. It looks to me like maybe the [json-script](https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#json-script) filter might allow this?

Reply:
Hmmm... I've been trying to get the json-script filter tag to work, but am currently getting an invalid GeoJSON error in console. WIP commit pushed [here](https://github.com/kevinearldenny/geodjango-vagrant-toy/commit/06c9471c0e357b99fd1dd6f24e436159fed0998a) if you wanna take a look

Reply:
Okay, I see what's going on here. The `data` variable is the output of `serialize()`, so it's actually already a string by this point, it doesn't need to be converted further. The filters `escapejs` and `json_script` are designed to operate on Python objects, so if they encounter a string they'll apply escaping to it so that it would work if it was to be embedded as a string literal within the Javascript. But that's not what we want to do -- we want to treat `data` as if it is JS source code already, rather than a string literal inside JS source code. 

I was able to get things working by doing `gj = {{ data|safe }}` (to prevent Django's default template escaping from applying) and then putting `gj` directly onto the map without parsing it first: `L.geoJSON(gj, ...` .